src/components/headers/collection-header.tsx

```diff
@@ -791,14 +910,26 @@ export class CollectionHeader extends React.Component<IProps, IState> {
       });
   };
 
+  private toggleImportModal(isOpen: boolean, warning?: string) {
+    const newAlerts = this.state.alerts;
+    if (warning) {
+      const newAlert = new AlertType();
+      newAlert.title = warning;
+      newAlert.variant = 'warning';
+      newAlerts.push();
+      this.setState({ alerts: newAlerts });
```

Comment:
pushing undefined to alerts..

```suggestion
      this.setState({ alerts: this.state.alerts.concat({
        title: warning,
        variant: 'warning',
      }) });
```